frontend/src/scenes/trends/ActionFilter/ActionFilterRow.js

```diff
@@ -32,8 +32,8 @@ export function ActionFilterRow({ filter, index, showMaths, typeKey }) {
         value = null
     } else {
         entity = entities[filter.type].filter(action => action.id == filter.id)[0] || {}
-        name = entity.name
-        value = entity.id
+        name = entity.name || filter.name
```

Comment:
extra condition because of the new logic @timgl added in action filter for initial filter